graph/src/components/store/traits.rs

```diff
@@ -156,8 +156,12 @@ pub trait SubgraphStore: Send + Sync + 'static {
 
     async fn is_healthy(&self, id: &DeploymentHash) -> Result<bool, StoreError>;
 
-    /// Find the deployment locators for the subgraph with the given hash
-    fn locators(&self, hash: &str) -> Result<Vec<DeploymentLocator>, StoreError>;
+    /// Find the deployment locators for the subgraph with the given hash.
+    /// If `only_active` is `true`, only the locator for the active
+    /// deployment will be returned. If it is `false`, locators for all
+    /// deployments with this hash will be returned.
+    fn locators(&self, hash: &str, only_active: bool)
```

Comment:
I think we should split this into something like `all_locators` and `active_locators` instead of using a flag, when someone is using or reading this function with a bool they need to go read the docs to figure what the bool is used for, when using all/active naming then it's obvious both when writing and when reading the code.

Reply:
I just removed the boolean flag and added a `fn active_locator` to the trait